testbed-managerless/environments/default.tfvars

```diff
@@ -7,8 +7,8 @@
 # override:nodes_boot_from_image
 flavor_manager            = "SCS-4V-8-50s"
 flavor_node               = "SCS-8V-32-100s"
-image                     = "Ubuntu 22.04"
-image_node                = "Ubuntu 22.04"
+image                     = "CentOS Stream 9"
+image_node                = "CentOS Stream 9"
 public                    = "public"
 availability_zone         = "nova"
 volume_availability_zone  = "nova"
```

Comment:
Use new image_user and image_node_user parameters to set cloud-user as username.